cloudbuild.yaml

```diff
@@ -4,4 +4,4 @@ steps:
     entrypoint: /bin/sh
     args:
       - -c
-      - "pip3 install -r requirements.txt && python3 -m pytest"
```

Comment:
Didn't realize this repo used pytest.  Is it in some way better than unittest?

Reply:
iirc from 2 years ago, it's better at handling module imports and discovery (we could say - find and run all files with *test.py from this dir). in the data repo though, we'd run into issues with how we're using FLAGS

Reply:
thanks.  both those are pretty painful today.  medium term wonder if it's worth switching data repo to pytest, or something better...

Reply:
Checking online, looks like pytest is more flexible, simple than unittest. It has many plugins which might be useful for the complicated case in data repo.